editoast/src/views/scenario/macro_nodes.rs

```diff
@@ -215,19 +215,21 @@ async fn create(
                     return Err(ScenarioError::NotFound { scenario_id }.into());
                 }
 
-                let macro_node = data
-                    .into_macro_node_changeset(scenario_id)?
-                    .create(&mut conn)
-                    .await?;
+                let changesets: Result<Vec<_>, _> = data
+                    .into_iter()
+                    .map(|node| node.into_macro_node_changeset(scenario_id))
+                    .collect();
```

Comment:
```suggestion
                let changesets = data
                    .into_iter()
                    .map(|node| node.into_macro_node_changeset(scenario_id))
                    .collect_vec();
```

there is no error path in `into_macro_node_changeset`, can you change it to return the changeste directly to simplify things a bit?

Reply:
Sure thing!